### PR TITLE
View skipped recipes

### LINKED ISSUE #317: Show in recipes explorer skipped recipes
Currently, when scanning the project the extension prevents to add in recipes explorer the recipe that have been skipped by Bitbake. 
As an example, if I want to add the the package `virt-manager` from `meta-virtualization` (https://git.yoctoproject.org/meta-virtualization/tree/recipes-extended/virt-manager/virt-manager_4.1.0.bb?h=master); it requires by default `wayland` or `x11`  distro features, and so will be skipped if the latter are not in the image being built:

```bash
pleguendekerneizon@ffde7735d379:~/PROJECTS/SEAPATH/seapath_yocto/build$ bitbake-layers show-recipes | grep virt-mana -C 5
  meta                 9.1.0114
vim-tiny:
  meta                 9.1.0114
virglrenderer:
  meta                 1.0.1 (skipped: missing required distro feature 'opengl' (not in DISTRO_FEATURES))
virt-manager:
  meta-virtualization  4.1.0 (skipped: one of 'x11 wayland' needs to be in DISTRO_FEATURES)
virt-viewer:
  meta-virtualization  11.0 (skipped: one of 'wayland x11' needs to be in DISTRO_FEATURES)
virt-what:
  meta-virtualization  1.21
```

The behavior of the extension is to not show at all the skipped recipe:
image

A nice feature would be to show the skipped recipe and display them maybe with a warning. The fact that recipe is not showing at all may lead to think a bug although this is an expected behavior.

### PR DESCRIPTION
**Pull request by deribaucourt**
Closes #317

### CODE REVIEW

**Comment by deribaucourt** [comment]
image

File: client/src/ui/BitbakeRecipesView.ts [hunk 1]
```diff
@@ -135,4 +146,8 @@ class BitbakeTreeDataProvider implements vscode.TreeDataProvider<BitbakeRecipeTr
     item.tooltip = 'Add a recipe to the active workspace'
     return item
   }
+
+  private getRecipeInfo (label: string): ElementInfo | undefined {
```
Comment (WilsonZiweiWang) [inline client/src/ui/BitbakeRecipesView.ts:158]: This is replaced by the `recipeInfoMap`, do we still need to keep it?

Comment (deribaucourt) [reply client/src/ui/BitbakeRecipesView.ts:158]: No longer used! good catch. Removing it.

**Review by WilsonZiweiWang — APPROVED** [review]
LGTM

**Pull request merged**